tracee-rules/engine/engine.go

```diff
@@ -59,29 +67,46 @@ func NewEngine(sigs []types.Signature, sources EventSources, output chan types.F
 			continue
 		}
 	}
-	return &engine
+	return &engine, nil
+}
+
+// signatureStartFunc is the signature handling business logics.
```

Comment:
why does this function name ends with `Func`? all functions are "Funcs" aren't they :) Can we change to `startSignature`?

Reply:
I guess I previously used it as a callback and forgot to rename :)